src/main.cpp

```diff
@@ -11,7 +11,17 @@
 
 using namespace std;
 
+void iPadTest();
```

Comment:
maybe move this stuff to another test.cpp or something?

Reply:
I think it would be better, if we finally create a common input interface, which we can use to hard code the files we want but can also be used with the command line interface you are building